ui/src/components/features/build-panel/components/files-dialog.tsx

```diff
@@ -96,105 +171,117 @@ export default function FilesDialog({ form, classNames }: TFilesDialog) {
-          <hr className="border-slate-400 pb-2" />
-          <DialogDescription>
-            <FormField
-              control={form.control}
-              name="file_ids"
-              render={({ field }) => {
-                return (
-                  <FormItem className="flex flex-col">
-                    <FormControl>
-                      <MultiSelect
-                        values={values}
-                        placecholder="Select a file..."
-                        defaultValues={formattedAssistantFiles}
-                        onValueChange={(selections) => {
-                          const fileIds = getFileIds(selections);
-                          field.onChange(fileIds);
-                        }}
-                      />
-                    </FormControl>
-                  </FormItem>
-                );
-              }}
-            />
-            <Card className="bg-slate-200">
-              <CardContent className="p-6 space-y-4">
-                <div className="border-2 border-dashed border-gray-700 rounded-lg flex flex-col gap-1 p-6 items-center">
-                  <FileIcon className="w-12 h-12" />
-                  <span className="text-sm font-medium text-gray-500">
-                    Drag and drop a file or click to browse
-                  </span>
-                </div>
-                <div className="space-y-2 text-sm">
-                  <Label>File upload</Label>
-                  <Input
-                    placeholder="Picture"
-                    type="file"
-                    accept="image/*, application/pdf"
-                    onChange={(event) => {
-                      handleFileChange(event);
-                    }}
-                  />
-                </div>
-              </CardContent>
-              <CardFooter>
-                <Button
-                  variant="default"
-                  disabled={!fileUpload}
-                  size="lg"
-                  type="button"
-                  onClick={handleUpload}
-                >
-                  Upload file
-                </Button>
-              </CardFooter>
-            </Card>
-          </DialogDescription>
-        </DialogHeader>
-        <DialogFooter>
-          <DialogClose asChild>
-            <Button size="lg" type="button">
-              Close
+        <Form {...form}>
+          <form
+            onSubmit={form.handleSubmit(onSubmit)}
+          >
+            <DialogHeader>
+              <DialogTitle className="mb-3 text-slate-300">
+                Manage Files
+              </DialogTitle>
+            </DialogHeader>
+            <DialogDescription>
+              <FormField
+                control={form.control}
+                name="files"
+                render={({ field }) => {
+                  return (
+                    <FormItem className="flex flex-col">
+                      <FormControl>
+                        <MultiSelect
+                          values={values}
```

Comment:
Should we hide this control when there are no values? otherwise, it was confusing. It said "Select files..." and when I clicked nothing was happening. Then I realised that it must be a list of uploaded/added files or something.

Reply:
Good idea! Added a brief text. Let me know what you think.
<img width="778" alt="Screenshot 2024-11-14 at 1 09 09 PM" src="https://github.com/user-attachments/assets/789c81f5-1616-41a5-9972-868c7b7740e5">